content/tutorial-nlp-from-scratch.md

```diff
@@ -575,7 +578,7 @@ def forward_prop(X_vec, parameters, input_dim):
         # Retrieve word corresponding to current time step
         x = X_vec[t]
         # Retrieve the embedding for the word and reshape it to make the LSTM happy
-        xt = emb_matrix.get(x, np.random.rand(input_dim, 1))
+        xt = emb_matrix.get(x, rng.standard_normal(size=(input_dim, 1)))
```

Comment:
This one was a `rand`, not `randn`, so I think it should be replaced with `rng.random(size=`

Reply:
Yes, you are right. Should be `rng.random(input_dim, 1)`